server/calendar/helpers.js

```diff
@@ -2,63 +2,76 @@
 
 const url = require('url');
 const entities = require('entities');
+const sanitizeHtml = require('sanitize-html');
 const unquote = require('unquote');
 
 const reDate = /^\d{1,2}\/\d{1,2}\d{4}$/;
 const reUrl = /url\((.*?)\)/;
 
 const baseUrl = 'http://infozona.hr';
 
+const textFilter = text => text.replace(/\n/g, '<br>');
+
+const sanitizeHTML = html => sanitizeHtml(html, textFilter);
+
 module.exports = {
-  readDate, readTime,
-  readInfo, readPhotoUrl, readLink
+  readDate, readTime, readInfo,
+  readPhotoUrl, readLink, sanitizeHTML
 };
 
 function readDate($date) {
   let date = $date.text().trim();
-  if (reDate.test(date)) return date;
-  return formatDate(new Date());
+  return reDate.test(date) ? normalizeHrDateString(date) : formatDate(new Date());
```

Comment:
Still looks bit clumsy 😢

Reply:
That's why we should use momentjs...
It has the [feature we need](http://momentjs.com/docs/#/parsing/string-format/).